### PR TITLE
common: Change linker variable to uint32.

### PR DESCRIPTION
**Pull request by kwagyeman**
Tested

### CODE REVIEW

**Comment by iabdalkader** [comment]
I actually realized that it's more work if it's uint32_t that's why I released without it.

**Comment by kwagyeman** [comment]
Close?

**Comment by iabdalkader** [comment]
Yup. Thanks though!

**Comment by iabdalkader** [comment]
Maybe it makes sense just for the MPU since it expects uint32_t ?

File: src/omv/common/mp_utils.c [hunk 1]
```diff
@@ -140,15 +140,15 @@ void mp_init_gc_stack(void *sstack, void *estack, void *heap_start, void *heap_e
     // Add GC blocks (if enabled).
     #ifdef OMV_GC_BLOCK1_MEMORY
     typedef struct {
-        uint8_t *addr;
+        uint32_t addr;
```
Comment (iabdalkader) [inline src/omv/common/mp_utils.c:143]: We just revert this then we're good.

Comment (kwagyeman) [reply src/omv/common/mp_utils.c:143]: k

Comment (kwagyeman) [reply src/omv/common/mp_utils.c]: Done

**Review by iabdalkader — APPROVED** [review]

**Pull request merged**